dasharo-security/secure-boot.robot

```diff
@@ -239,10 +239,7 @@ SBO009.001 Attempt to boot file signed for intermediate certificate
     Skip If    not ${SECURE_BOOT_SUPPORT}    SBO009.001 not supported
     Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    SBO009.001 not supported
     Skip If    not ${TESTS_IN_UBUNTU_SUPPORT}    SBO009.001 not supported
-    Download ISO And Mount As USB
-    ...    ${DL_CACHE_DIR}/${INTERMEDIATE_NAME}
-    ...    ${INTERMEDIATE_URL}
-    ...    ${INTERMEDIATE_SHA256}
+    Make ISO And Mount As USB    intermediate
```

Comment:
@WiktorG351 this could go to suite setup so we would need to run it once, before running tests, this way we would prepare all needed images, additionally we could add variable so by default those images would not be created (user would need to force creation only once), something similar is here https://github.com/Dasharo/open-source-firmware-validation/pull/193/files

Reply:
@TomaszAIR what if someone is only running one of these tests? Then it would be unnecessary to create all the other images as part of suite setup.

Maybe since the scripts require `sudo` anyway, one solution I can think of is this:
* the repo by default does not have the images
* the tests themselves dont generate the images, but require them
* if you run the tests, and dont have the images, they will fail and tell you that you need to use the script to generate them manually
* then the tests can run
that way we will the generation (and sudo) out of the RF code, and it will be super clean.
Just one keyword like `Mount Image As USB` or something.

Reply:
This is better approach, we should just add needed information in README how to prepare env to run test (e.g. run script before running tests)